java/src/main/java/com/adhoc/flight/QueryRunner.java

```diff
@@ -120,37 +120,35 @@ public static void runDemo() throws Exception {
 
         // Authenticates FlightClient with routing properties.
         try (final AdhocFlightClient client = createFlightClient(routingCallOption)) {
-            PrintRowProcessor rowProcessor = new PrintRowProcessor();
-            rowProcessor.printAuthenticated(ARGUMENTS.host, ARGUMENTS.port);
+            BaseTinyIntVectorProcessor.printAuthenticated(ARGUMENTS.host, ARGUMENTS.port);
 
             /**
              * Create demo table in $scratch
              */
             System.out.println("[INFO] [STEP 2]: Create a test table in $scratch.");
-            client.runQuery(CREATE_DEMO_TABLE, null, null);
+            client.runQuery(CREATE_DEMO_TABLE);
             System.out.println("[INFO] Created $scratch.dremio_flight_demo_table in $scratch successfully.");
 
             /**
              * Query demo table
              */
             System.out.println("[INFO] [STEP 3]: Query demo table $scrach.dremio_flight_demo_table");
```

Comment:
Unrelated typo: $scrach -> $scratch.

Reply:
I will make sure to correct this in a future refactor PR.